runner/src/check_todo.rs

```diff
@@ -15,16 +15,48 @@
 //
 
 use super::*;
+use regex::Regex;
+use std::cmp::max;
 
 /// A [`Runnable`] command that checks for the existence of todos in the codebase with no associated
 /// GitHub issue number.
 pub struct CheckTodo {
     path: String,
+    todo_pattern: Regex,
 }
 
 impl CheckTodo {
     pub fn new(path: String) -> Box<Self> {
-        Box::new(CheckTodo { path })
+        Box::new(CheckTodo {
+            path,
+            todo_pattern: Regex::new(r"TODO\(#\d+\)").expect("Could not parse regex"),
+        })
+    }
+
+    fn is_invalid_todo_line(&self, line: &str) -> bool {
+        line.split_whitespace()
+            .filter(|word| word.contains("TODO"))
+            .any(|word| !self.todo_pattern.is_match(word))
+    }
+
+    fn get_comment_lines(file_content: String) -> Vec<String> {
```

Comment:
@juliettepretot (I cannot reply to your original comment about multiline comments any more!) 
I've added this function to include multiline comments in the check for `TODO`s. Let me know if there are any important cases that I am missing. Or if you can think of a way to simplify this function :)

Reply:
It seem good to me. I kind of do wonder if we really need the `comment_line` check at all though. At this point we're basically implementing our own code parser, that parses files for comments and even supports nested comments. It's nice engineering, but I suspect some of my past colleagues would've called it over-engineered. ;) 

The original code checked for for `"TODO"` regardless of whether it lives it in a comment, and it seemed like that worked fine? 

I personally wouldn't add comment-parsing unless we really need it, but I also don't have a strong opinion on it. :)

Reply:
Yes. I agree. I am not sure about keeping this part either, because of the exact same reasons :) 
I'll change it back to checking for TODOs in every line.